### PR TITLE
Add Biordle to the list.

### PR DESCRIPTION
**Pull request by Brayman30**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -197,6 +197,7 @@ For mobile and wearable devices and smart TVs, sharing of results is initially a
 ### Math, Acronyms, Science, Tech, and more
 
 - [AI-powered](https://github.com/asirota/wordle-ai): Includes an AI component
+- [Biordle](https://github.com/Brayman30/biordle): Biology
```
Comment (igitur) [inline README.md:200]: I see many of the other links point to the repos, but the intention is actually to point to your actual implementation, i.e. https://biordle.grenteam.com/

Comment (igitur) [inline README.md:200]:
```suggestion
- [Biordle](https://biordle.grenteam.com): Biology
```

**Comment by igitur** [comment]
Is Biordle still working? If not, please remove from the README.

**Comment by igitur** [comment]
Disappointed in your lack of response, @Brayman30

**Comment by igitur** [comment]
Removed Biordle from README

**Comment by Brayman30** [comment]
> Disappointed in your lack of response, @Brayman30 

Sorry that I missed this, I have not had time to work on open source over the past few months for personal and school reasons. Biordle is no longer maintained and I should have marked it as such. I will be doing so in the coming days.

**Pull request merged**